js/config.js

```diff
@@ -1,13 +1,14 @@
 module.exports = {
   output: './test/generic',
   ext: '.spec.js',
-  emitter: 'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/master/js/emitter.js',
+  emitter: 'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/universal-sdk/js/emitter.js',
   overrides: [
-    'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/master/js/overrides.js',
-    ...(process.env.APPLITOOLS_TEST_REMOTE === 'eg' ? ['https://raw.githubusercontent.com/applitools/sdk.coverage.tests/master/eg.overrides.js'] : []),
+    'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/universal-sdk/js/overrides.js',
+    ...(process.env.APPLITOOLS_TEST_REMOTE === 'eg' ? ['https://raw.githubusercontent.com/applitools/sdk.coverage.tests/universal-sdk/eg.overrides.js'] : []),
   ],
   template:
-    'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/master/js/template.hbs',
+    'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/universal-sdk/js/template.hbs',
+  tests: 'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/universal-sdk/coverage-tests.js',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
  tests: "https://raw.githubusercontent.com/applitools/sdk.coverage.tests/universal-sdk/coverage-tests.js",
```